spec/functional_tests/people/create_person_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+user = TestingAppServer::UserData.new(first_name: 'John', last_name: 'Smith',
+                                      mail: TestingAppServer::PrivateData.new.decrypt['user_mail'],
+                                      pwd: SecureRandom.hex(7), type: :user)
```

Comment:
I would setup like `14` and it's enought

Reply:
The resulting hexadecimal string is twice `n` (https://apidock.com/ruby/SecureRandom/hex/class), so now we are already generating 14 digits